docs/download_files.md

```diff
@@ -139,6 +139,40 @@ For more on the specific contents of multiplexed library `SingleCellExperiment`
 
 The [metadata file](#metadata) for multiplexed libraries (`single_cell_metadata.tsv`) will have the same format as for individual samples, but each row will represent a particular sample/library pair, meaning that there may be multiple rows for each `scpca_library_id`, one for each `scpca_sample_id` within that library.
 
+
+## Merged object downloads
+
+Merged object downloads contain all single-cell or single-nuclei gene expression data for a given ScPCA project within a single object, provided as either a [`SingleCellExperiment` object (`.rds` file)](http://bioconductor.org/books/3.13/OSCA.intro/the-singlecellexperiment-class.html) or an [`AnnData` object (`.hdf5` file)](https://anndata.readthedocs.io/en/latest/index.html).
```

Comment:
I would add a sentence here first that says something like: 

"If downloading all samples associated with a project, data can be downloaded so that each sample is in an individual file or all data is in a single file or merged object."

I don't think the wording is quite right, but this is the message I would portray so people understand what use case they would even want a merged object.